src/Database/PostgreSQL/PQTypes/SQL/Builder.hs

```diff
@@ -291,10 +295,20 @@ data SqlDelete = SqlDelete
   , sqlDeleteRecursiveWith :: Recursive
   }
 
--- | This is not exported and is used as an implementation detail in
--- 'sqlWhereAll'.
-newtype SqlAll = SqlAll
-  { sqlAllWhere :: [SqlCondition]
+
+-- | Type representing a set of conditions that are joined by 'AND'.
+--
+-- This is usually not needed as the default behavior is to join conditions
+-- by 'AND', but it is used to implement `sqlWhereAny`.
```

Comment:
Is it?

Reply:
I guess I'll remove the implementation details from the comments.